templates/articles/article_index_page.html

```diff
@@ -14,7 +14,7 @@
         <h2 class="sr-only">{% trans "Each collection insight takes an in-depth look into a group of the records we hold." %}</h2>
     </div>
 
-    {% if page.featured_article %}
+    {% if page.featured_article.live %}
```

Comment:
> or make it private - then view the original page where you added the featured article and see that it is missing.

When its made private, its still live and can be seen. 

You might want to check a method [get_view_restrictions().exists()](https://github.com/wagtail/wagtail/blob/main/wagtail/models/__init__.py#L2474-L2482) if it exists or not - maybe something as a template tag or so?

Reply:
Hey @TNA-Allan, I had a think about this, and I think just having `.live` will be fine. The ticket only asks for unpublished pages, and I think it should be down to editors to not put a page in the featured article field if they're going to make it private at some point... I'm happy with this code as is, so if you're happy with it too then this is done 😃

Reply:
@jamesbiggs , I going with the view you have taken.